scripts/updateSources.js

```diff
@@ -216,22 +156,9 @@ async function main() {
     // charts v14
     content = content.replace('getTestView().getCanonicalName();', '"vaadin-charts/" + getTestView().getCanonicalName();');
 
-    content = content.replace('import com.vaadin.flow.demo.ComponentDemoTest','import com.vaadin.tests.ComponentDemoTest');
-    content = content.replace('import com.vaadin.flow.demo.TabbedComponentDemoTest','import com.vaadin.tests.TabbedComponentDemoTest');
-    content = content.replace('import com.vaadin.testbench.parallel.ParallelTest','import com.vaadin.tests.ParallelTest');
-    content = content.replace('import com.vaadin.flow.testutil.AbstractComponentIT','import com.vaadin.tests.AbstractComponentIT');
-    content = content.replace('import com.vaadin.flow.testutil.AbstractValidationTest','import com.vaadin.tests.AbstractValidationTest');
     content = content.replace('import com.vaadin.testbench.annotations.BrowserConfiguration;','');
     content = content.replace(/.*@BrowserConfiguration.*/,'');
 
-    // Remove W3C workaround from Grid tests.
```

Comment:
The original workaround was removed in https://github.com/vaadin/vaadin-grid-flow/pull/1101.